templates/workflow/submissionHeader.tpl

```diff
@@ -25,7 +25,7 @@
 		</h1>
 		<div class="pkp_submission_author">
 			<span class="pkp_screen_reader">{translate key="user.role.author_s"}</span>
-			{$submission->getAuthorString()}
+			{$submission->getAuthorString(false, ', ', '; ', $newestVersion)}
```

Comment:
The same as above: If that default version = 1 in Submission::getAuthors would get the latest submission version this would eventually not be necessary?